pkg/controller/secret/secret_controller.go

```diff
@@ -317,20 +315,30 @@ func createOCMAgentReceiver(ocmAgentURL string) []*alertmanager.Receiver {
 
 // createPagerdutyConfig creates an AlertManager PagerdutyConfig for PagerDuty in memory.
 func createPagerdutyConfig(pagerdutyRoutingKey, clusterID string) *alertmanager.PagerdutyConfig {
+	detailsMap := map[string]string{
+		"alert_name":   `{{ .CommonLabels.alertname }}`,
+		"link":         `{{ if .CommonAnnotations.runbook_url }}{{ .CommonAnnotations.runbook_url }}{{ else if .CommonAnnotations.link }}{{ .CommonAnnotations.link }}{{ else }}https://github.com/openshift/ops-sop/tree/master/v4/alerts/{{ .CommonLabels.alertname }}.md{{ end }}`,
+		"ocm_link":     fmt.Sprintf("https://console.redhat.com/openshift/details/%s", clusterID),
+		"num_firing":   `{{ .Alerts.Firing | len }}`,
+		"num_resolved": `{{ .Alerts.Resolved | len }}`,
+		"resolved":     `{{ template "pagerduty.default.instances" .Alerts.Resolved }}`,
+		"cluster_id":   clusterID,
+	}
+
+	if config.IsFedramp() {
```

Comment:
Going with my above comment, there could be one template for standard clusters and one template for fedramp clusters.

Reply:
No I mean a different file. Then if fedramp, load the fedramp version of the file